docs/contributing/core/index.md

```diff
@@ -0,0 +1,268 @@
+If you do not know, you can ask in the [Plone Community Forum](https://community.plone.org/).
+Only a few packages are in {file}`src/` by default.
+
+Edit the file {file}`checkouts.cfg` at the root of the repository by adding the package names under the `auto-checkout` list.
+
+```{code-block} cfg
+:emphasize-lines: 10-
+[buildout]
+always-checkout = force
+# always-checkout = false
+auto-checkout =
+# These packages are always checked out:
+    docs
+    # <snip>
+# These packages are manually added, or automatically added by mr.roboto:
+    # <snip>
+    plone.app.event
+    icalendar
+```
+
+Then rerun buildout to install the source packages to edit.
+
+```shell
+./bin/buildout
+```
+
+```{seealso}
+For tips on working with `mr.developer`, see {doc}`mrdeveloper`.
+```
+
+
+## Test locally
+
+If you change the expected behavior of a feature in a package, you should write a test to cover the change.
+
+To run a test for the specific package that you modified, use the `-m` option followed by the package name, as shown in the following example.
+
+```shell
+./bin/test -m plone.app.caching
+```
+
+If any test fails, do not commit and push the changes.
+Instead write a test that passes.
+
+After the package level tests pass with your change, you can run the full unit test suite to ensure other packages aren't affected by the change.
+It takes 5-10 minutes to run the full unit test suite.
+
+```shell
+# Run unit tests
+./bin/test
+```
+
+If you run acceptance tests with the `--all` option, it will repeatedly launch and close browser windows that gain focus, disrupting you from doing any other work.
+This takes 30-40 minutes to run.
+If you can't afford this disruption, you can defer it to Jenkins.
+
+```shell
+# Run acceptance tests
+./bin/test --all
+```
+
+
+## Change log
+
+All changes require a change log entry.
+
+For packages that use [towncrier](https://pypi.org/project/towncrier/) to produce change logs, see {ref}`contributing-change-log-label`.
+
+For packages that don't use towncrier, edit either {file}`CHANGES.rst`, {file}`CHANGES.txt`, or {file}`HISTORY.txt` in each package you have modified, adding a summary of the change.
+New change log entries should be added at the top of {file}`CHANGES.rst`.
+
+
+## Update `checkouts.cfg`
+
+If you didn't do so earlier, edit the file {file}`checkouts.cfg` at the root, and add your changed packages only to the `auto-checkout` list.
+Remove any packages that you previously added to this file, but did not change.
+Leave any pre-existing packages in the original list.
+
+The Plone release manager will check this file to see which packages you have updated to make a proper release.
+
+```{seealso}
+{doc}`release`
+```
+
+
+## Create a pull request
+
+After you have completed all the foregoing steps, push your changes to a remote branch and create a pull request in GitHub.
+
+If you are working from an issue, return to the original issue, and add a link to the pull request.
```

Comment:
Or mention the issue as "Fixes #issuenum" in the PR description, and github will create the link

Reply:
Done in bc12737979848953a97e6aa2b9e18d236171b14e